src/modules/configs/config.py

```diff
@@ -25,6 +28,14 @@ def validate_json(conf: dict, schema_file) -> None:
             from ex
 
 
+class ConfigRulesError(Exception):
```

Comment:
Shouldn't it actually `raise Exception` instead of `pass`?

Reply:
nope, i refer to this article.  and it works fine
[https://realpython.com/python-raise-exception/#choosing-the-exception-to-raise-built-in-vs-custom](url)